.github/workflows/tf-account.yaml

```diff
@@ -12,10 +12,17 @@ on:
         required: true
       TF_BACKEND_S3_REGION:
         required: true
+    inputs:
+      TERRAFORM_VERSION:
+        required: false
+        type: string
+
+env:
```

Comment:
Remove this indirection, set a `default` for the input

https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs